README.md

```diff
@@ -1 +1,34 @@
-# school_library
\ No newline at end of file
+# school_library
+ > The school Library is an a tool that records what books are in the library and who borrowed them. This app will allow you to:
+
+ - Add new students or teachers.
+ - Add new books.
+ - Save records of who borrowed a given book and when.
+And all of this will be built in a beautiful and well-organized way!
+
+## Authors
+
```

Comment:
- [OPTIONAL] Please it will be nice to add a `Getting started` section to your `Readme file`, where you outline explicit steps on how to clone and run your app locally, as this will be of help to non technical users. Please always make use of this [template](https://github.com/microverseinc/readme-template) from Microverse and adapt it to your project as this will make your readme file more informative and professioanal.